src/intro/tooling.md

```diff
@@ -15,15 +15,13 @@ tested.
 - OpenOCD >=0.8. Tested versions: v0.9.0 and v0.10.0
 - GDB with ARM support. Version 7.12 or newer highly recommended. Tested
   versions: 7.10, 7.11, 7.12 and 8.1
-- [OPTIONAL] `git` OR
-  [`cargo-generate`](https://github.com/ashleygwilliams/cargo-generate). If you
-  have neither installed then don't worry about installing either.
+- [`cargo-generate`](https://github.com/ashleygwilliams/cargo-generate) and `git`.
+These tools are optional but will make it easier to follow along with the book.
 
 The text below explains why we are using these tools. Installation instructions
 can be found on the next page.
 
-## `cargo-generate` OR `git`
-
+## Using a template for the first project ##
```

Comment:
All headings are named after tools. With this change that would no longer be the case. I would prefer to keep the symmetry.

Reply:
What about `cargo-generate and other tools` ?

Reply:
By the way, maybe write something about rustc too ..

Reply:
The current text matches what's listed in the bullet list. The "other tools" in your proposal makes it unclear whether "other tools" means just git or all / some of the other tools in the bullet list.

Personally, I think the current text is fine.

Reply:
> By the way, maybe write something about rustc too ..

Rust developers (the target audience of this book) are familiar with rustc / Cargo; I don't think it's necessary to add a section for them.

Reply:
i just find it strange that the choice now is between `cargo generate` and `git` both tools need to be downloaded and installed, while the other tools "curl, wget, or your web browser." are likely to be already on a system. But i'm tired to bike shed over this, so you can get the final call, i'll leave it as it is or make it whatever.